foyer/general_forcefield.py

```diff
@@ -34,17 +36,18 @@ def preprocess_forcefield_files(forcefield_files=None, backend="gmso"):
             _, suffix = os.path.split(file)
             tempfile = NamedTemporaryFile(suffix=suffix, delete=False)
             try:
+                Validator(ff_file_name=file, debug=False)
                 from_foyer_xml(
                     foyer_xml=str(file),
                     gmso_xml=str(tempfile.name),
                     overwrite=True,
                 )
-                tmp_processed_files.append(tempfile.name)
-            except Exception:
+            except:
                 warnings.warn(
                     f"Could not convert {str(file)}, attempt to read in as is."
```

Comment:
This warning could be more descriptive because I think it's one that might be raised commonly if you're already passing gmso `.xmls`. I think something like:

f"Could not convert {str(file)} from a foyer xml, attempting to read in as gmso xml schema."

would suffice

Reply:
I have updated the warning as suggested.